README.md

```diff
@@ -61,6 +61,59 @@ Table of Contents
 
 ## Things to be aware of when writing code
 
+###Writing secure software
```

Comment:
These added sections have too many slashes, e.g., "Use/Implement tests/build processes/libraries", "protect you/ sensitive information", "what you click/attach" etc.  Spell out "and" and "or" properly to make the text flow better.